docroot/modules/custom/annotation_store/src/Form/AnnotationStoreConfigurationForm.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Drupal\annotation_store\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Class SettingsForm.
+ *
+ * @package Drupal\annotation_store\Form
+ */
+class AnnotationStoreConfigurationForm extends ConfigFormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditableConfigNames() {
+    return [
+      'annotation_store.settings',
+    ];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'settings_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $config = $this->config('annotation_store.settings');
+    $form['annotation_store_date_format'] = array(
```

Comment:
Why do we need this configuration? Related to comment on Entity's buildRow() function.